feature/page_objects/base_page.py

```diff
@@ -1,7 +1,10 @@
 from cgitb import text
+from lib2to3.pgen2 import driver
```

Comment:
delete this 2 unused imports
`from cgitb import text`
`from lib2to3.pgen2 import driver`